cypress/support/commands.js

```diff
@@ -47,4 +47,13 @@ Cypress.Commands.add('login', (email, password) => {
   cy.get('.modal-footer > .btn-primary').should('be.enabled').click();
   cy.url().should('contain', '/panel/garage');
   cy.get('.alert-success').should('contain', 'You have been successfully logged in');
+});
+
+Cypress.Commands.add('loginAlt', (email1, password1, email2, password2) => {
```

Comment:
overall everything is OK, I'm not happy with this, if you have 3 envs to support ?) 4?) 
Wouldn't it be easier to either store creds in config and based on URL put them into cypress.env? or, if not a good idea, bypass them as other env variables, like base URL?

Reply:
fixed it now Teacher :)

Reply:
in my updated pull request: https://github.com/artqa/aqa-advanced/pull/28/files